src/IndexingWrapper.cpp

```diff
@@ -97,10 +97,14 @@ parseField(const string & field, const char & separator)
 //'
 //' @param dbpath	path to a Xapian database
 //' @param dataFrame	data frame to be indexed
-//' @param idColumn	index of a column in the data frame whose row value will be used as a unique identifier to each document(row)
-//' @param indexFields	list of 'list of fields' that will be indexed using a Xapian::TermGenerator
-//' @param filterFields	list with information on boolean terms to be added to Xapian::Document
-//' @param stemmer 	the stemmer that should be applied to the Xapian::TermGenerator
+//' @param idColumn	index of a column in the data frame whose row value
+//will be used as a unique identifier to each document(row)
```

Comment:
I think this is going to be one of those times where you have to wrap by hand to preserve the `//' ` prefix properly. If you do that and indent the wrapped lines (so "will" aligned with "index") then clang-format shouldn't re-wrap it.